doc/source/theory/dem/dem.rst

```diff
@@ -121,54 +127,89 @@ Where:
 
 * :math:`\nu_i, \nu_j` poisson coefficient of particle i or j;
 
-Rolling friction may be calculated through a constant torque model or a viscous torque model corresponding to those equations:
+~~~~~~~~~~~~~~~~~~~~
+Coulomb's limit
+~~~~~~~~~~~~~~~~~~~~
+
+Coulomb's criterion is violated when the following condition is not respected during a collision:
 
 .. math::
-    \mathbf{M}_{ij}^{r} &= -\mu_{r}R_{e}|\mathbf{F}_{ij}^{n}| \mathbf{\hat{\omega}}_{ij} \\
-    \mathbf{M}_{ij}^{r} &= -\mu_{r}R_{e}|\mathbf{F}_{ij}^{n}||\mathbf{V}_{\omega}| \mathbf{\hat{\omega}}_{ij}
+    |\mathbf{F}_{ij}^{t}| \geq \mu |\mathbf{F}_{ij}^{n}|
 
-Where the parameters are:
+
+A violation means the collision is having gross sliding and tangential force needs to be limited to the Coulomb limit.
+To do so, the tangential overlap :math:`\mathbf{\delta_t}` is first limited and then the tangential force is recalculated.
+
+The tangential overlap is calculated with the tangential force with no damping force as default nonlinear contact model as follow:
 
 .. math::
-    \mathbf{\hat{\omega}}_{ij} &= \frac{\omega_{i} - \omega_{j}}{|\omega_{i} - \omega_{j}|} \\
-    \mathbf{V}_{\omega} &= \left( \omega_{i} \times R_{i}\mathbf{n}_{ij}-\omega_{j} \times R_{j}\mathbf{n}_{ji} \right)
+    \mathbf{\delta_t} &= \frac{\mathbf{\tilde{F}_{ij}}}{-k_{t}} \\
+    \mathbf{\tilde{F}_{ij}} &= \mathbf{\hat{F}_{ij}} + \eta_{t}\mathbf{v}_{rt} \\
+    \mathbf{\hat{F}_{ij}^{t}} &= \mu |\mathbf{F}_{ij}^{n}| \frac{\mathbf{F}_{ij}^{t}}{|\mathbf{F}_{ij}^{t}|}
 
-Where:
+Regarding the particle-wall contacts, applied models are the same than particle-particle contacts with a background triangulation and mapping with walls.
 
-* :math:`\mu_{r}` rolling friction coefficient;
+.. note::
+    When using a cohesive force models, Coulomb's criterion need to be adjusted, since the normal contact force isn't equal to the total normal force between particle in a collision.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Tangential torque
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Tangential torque is calculated through:
+Tangential torque is the torque generated by the tangential force. It can be calculated through:
 
 .. math::
-    \mathbf{M}_{ij}^{t} = R_{i}\mathbf{n}_{ij} \times \mathbf{F}_{ij}^{c}
+    \mathbf{M}_{ij}^{t} = R_{i}\mathbf{n}_{ij} \times \mathbf{F}_{ij}^{t}
 
-Coulomb's criterion is violated when this condition is not respected during a collision:
+.. note::
+    As of February 2024, the lethe-particles solver only uses spherical particles, thus the normal force does not generate a torque on the particle during a collision.
 
-.. math::
-    |\mathbf{F}_{ij}^{t}| \geq \mu |\mathbf{F}_{ij}^{n}|
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Rolling friction models
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Rolling friction may be calculated through a constant torque model or a viscous torque model. The user also has the option of not using a rolling friction model. The corresponding model can be described by the following equations:
```

Comment:
Since this is the Theory guide, I would not mention "the user" or options. This is more appropriate for the Parameters guide